source/logging-monitoring/change-streams.txt

```diff
@@ -51,16 +61,79 @@ To open a change stream, call the ``watch()`` method on an instance of a
    see the :ref:`<replica-set-oplog>` {+mdb-server+} manual page.
 
 The object on which you call the ``watch()`` method on determines the scope of
-events that the change stream listens for.
+events that the change stream listens for:
+
+- ``MongoCollection.watch()`` monitors a collection.
+- ``MongoDatabase.watch()`` monitors all collections in a database.
+- ``MongoClient.watch()`` monitors all changes in the connected MongoDB deployment.
+
+Filter the Events
+~~~~~~~~~~~~~~~~~
+
+The ``watch()`` method optionally takes an **aggregation pipeline**  which
```

Comment:
This still doesn't read well to my ears.  To many "which"s.  Can you try to re-word it?